axelor-account/src/main/resources/domains/Tax.xml

```diff
@@ -7,6 +7,7 @@
 
   <entity name="Tax" cacheable="true">
 
+    <boolean name="manageByAmount" title="Tax amount management type"/>
     <boolean name="isNonDeductibleTax" title="Non deductible tax"/>
```

Comment:
why adding this field? It's already in base module
And you have to remove isNonDeductibleTax because you added it in base module